plugins/modules/site_workflow_manager.py

```diff
@@ -125,9 +125,10 @@
                 description: |
                     Floor number within the building site (e.g., 5). This value can only be specified during the creation of the
                     floor and cannot be modified afterward.
```

Comment:
It is required from version 2.3.7.6 onwards.

Reply:
Modified